release.json

```diff
@@ -22,6 +22,24 @@
         "WINDOWS_DDNPM_SHASUM": "61e13fccdca7bf6a356e7ef552a7845316b9905b5eee264811982e3034a3bb84",
         "SECURITY_AGENT_POLICIES_VERSION": "master"
     },
+    "6.28.0-rc.3": {
+        "INTEGRATIONS_CORE_VERSION": "7.28.0-rc.2",
+        "OMNIBUS_SOFTWARE_VERSION": "7.28.0-rc.1",
+        "OMNIBUS_RUBY_VERSION": "7.28.0-rc.1",
+        "JMXFETCH_VERSION": "0.43.0",
+        "JMXFETCH_HASH": "2aaf3338313c068b02addbf67e3dcad480c0ec78d668bc067ee519bc736613be",
+        "SECURITY_AGENT_POLICIES_VERSION": "v0.9",
+        "MACOS_BUILD_VERSION": "6.28.0-rc.3"
+    },
+    "7.28.0-rc.3": {
+        "INTEGRATIONS_CORE_VERSION": "7.28.0-rc.2",
+        "OMNIBUS_SOFTWARE_VERSION": "7.28.0-rc.1",
+        "OMNIBUS_RUBY_VERSION": "7.28.0-rc.1",
+        "JMXFETCH_VERSION": "0.43.0",
+        "JMXFETCH_HASH": "2aaf3338313c068b02addbf67e3dcad480c0ec78d668bc067ee519bc736613be",
+        "SECURITY_AGENT_POLICIES_VERSION": "v0.9",
+        "MACOS_BUILD_VERSION": "7.28.0-rc.3"
```

Comment:
It seems like the `WINDOWS_DDNPM` entries are missing (same for `6.28.0-rc.3`)?

Reply:
Hrmm... great catch - fixing.

Reply:
Fixed